app/views/metrics/index.html.erb

```diff
@@ -87,7 +87,7 @@
               </div>
               <div class="results"></div>
             </div>
-          <h5> Archived students </h5> <b>Resolved and contacted students becomes archived when student metrics are changed </b>
+          <h5> Archived students </h5> <b>Resolved and contacted students becomes archived when student metrics are changed OR when dropped from the course </b>
```

Comment:
previous typo from before but: "Resolved and contacted students **become** archived when student metrics are changed OR when dropped from the course"